src/System.Text.Json/tests/Resources/Payloads/BlogPost.json

```diff
@@ -0,0 +1,996 @@
+﻿{
```

Comment:
Consider moving these into https://github.com/dotnet/corefx-testdata (an established pattern) if they are particularly large. Not sure what our usual threshold is. All summing to 50K maybe?